sa/sa_test.go

```diff
@@ -769,6 +770,7 @@ func TestAddIssuedNames(t *testing.T) {
 func TestDeactivateAuthorization(t *testing.T) {
 	sa, _, cleanUp := initSA(t)
 	defer cleanUp()
+	features.Set(map[string]bool{"AllowAccountDeactivation": true})
```

Comment:
Blergh, you're right it is in the wrong place. Since I also forgot to add a `defer features.Reset()` though it mean't those features were still enabled when we got to `TestDeactivateAuthorization`  :/